tests/cli/commands/test_build.py

```diff
@@ -61,12 +63,17 @@ def test_run_build_with_plan_should_execute_successfully(self):
         run_properties = RUN_PROPERTIES
 
         projects = [get_minimal_project()]
-        run_plan = {Stage.BUILD: projects, Stage.TEST: projects, Stage.DEPLOY: projects}
+        run_plan = {
+            TestStage.build(): projects,
+            TestStage.test(): projects,
+            TestStage.deploy(): projects,
```

Comment:
Not related to this pr, but shouldn't we add postdeploy to `TestStage` for this test and `tests/reporting/__init__.py` below?

Reply:
What would be the goal? 
I think this emphasises that not every pipeline will use all of the stages.